source/frontend/src/features/mapSideBar/compensation/models/CompensationRequisitionFormModel.ts

```diff
@@ -58,7 +58,10 @@ export class CompensationRequisitionFormModel {
 
   toApi(): ApiGen_Concepts_CompensationRequisition {
     const compReqPayees =
-      this.payees?.map<ApiGen_Concepts_CompReqPayee>(formPayee => formPayee.toApi()) ?? [];
+      this.payees
+        ?.map<ApiGen_Concepts_CompReqPayee>(formPayee => formPayee.toApi())
+        ?.filter(exists) ?? [];
+
```

Comment:
We don't want to save `CompReqPayee` where all the payees are null - so we filter those records before calling the server